aisynphys/cell_class.py

```diff
@@ -248,8 +250,14 @@ def classify_cells(cell_classes, cells=None, pairs=None):
     cell_groups = OrderedDict([(cell_class, set()) for cell_class in cell_classes])
     for cell in cells:
         for cell_class in cell_classes:
-            if cell in cell_class:
-                cell_groups[cell_class].add(cell)
+            if missing_attr is 'ignore':
+                try:
+                    if cell in cell_class:
+                        cell_groups[cell_class].add(cell)
+                except: Exception
+            else:
+                if cell in cell_class:
+                    cell_groups[cell_class].add(cell)
```

Comment:
I suggest rearranging the logic here to avoid repeating code:

    try:
        ...
    except Exception:
        if missing_attr == 'ignore':
            ...
        else:
            raise